### PR TITLE
Bug 1156225 - Update layout to match spec. r=drs

### PR DESCRIPTION
**Pull request by daleharvey**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -8,10 +8,10 @@
 
     <title>Bugzilla Lite</title>
 
-    <link rel="icon" href="/style/icons/app-128.png" sizes="128x128" type="image/png">
-    <link rel="icon" href="/style/icons/app-32.png" sizes="32x32" type="image/png">
+    <link rel="icon" href="/style/icons/bzlite_84.png" sizes="84x84" type="image/png">
     <link rel="stylesheet" href="/style/bzlite.css">
-    <link rel="stylesheet" media="(min-width: 800px)" href="/style/bzlite-desktop.css">
+    <!-- <link rel="stylesheet" media="(min-width: 800px)"
+               href="/style/bzlite-desktop.css"> -->
```
Comment (DouglasSherk) [inline public/index.html]: Should probably just remove this.

File: public/style/bzlite.css [hunk 2]
```diff
@@ -324,7 +370,7 @@ header a {
 }
 
 .comments li:nth-child(even) {
-  background-color: #F6F6F6;
+  background-color: #FFF;
```
Comment (DouglasSherk) [inline public/style/bzlite.css]: ```white``` .

File: public/views/bug-comments.tpl [hunk 3]
```diff
@@ -19,7 +18,7 @@
       </select>
     </div>
     <textarea placeholder="Reply..." id="commentInput"></textarea>
-    <input type="text" placeholder="Need More Information" id="needinfo" />
+    <input type="text" placeholder="Need More Information" id="needinfo" style="display: none" />
```
Comment (DouglasSherk) [inline public/views/bug-comments.tpl]: Should use a ```hide``` class or some such here instead.

File: public/views/profile.tpl [hunk 4]
```diff
@@ -1,3 +1,12 @@
...
-  <a href="/logout/" class="btn warning">Logout</a>
+<div>
+
+  <header>
+    <a href="/" class="headerBtn close"></a>
+    <h2>Profile</h2>
+    <span class="headerBtn"></span>
+  </header>
+
+  <div style="padding: 1rem">
```
Comment (DouglasSherk) [inline public/views/profile.tpl]: This should use a class instead.

File: public/style/bzlite.css (hunk 2 above)
Comment (daleharvey) [inline public/style/bzlite.css]: Yup, the background behind is light grey (this is a zebra strip), cheers

**Pull request closed without merging**